verifier/src/main/kotlin/itmo/verifier/formula/Formula.kt

```diff
@@ -4,22 +4,33 @@ import com.github.h0tk3y.betterParse.grammar.parser
 import com.github.h0tk3y.betterParse.lexer.literalToken
 import com.github.h0tk3y.betterParse.lexer.regexToken
 import com.github.h0tk3y.betterParse.parser.Parser
+import itmo.verifier.model.Variable
 
 sealed class CTLFormula {
     abstract fun optimize(): CTLFormula
-
+    open fun compute(elements: Map<String, Variable>): Boolean {
+        return true
+    }
```

Comment:
Should be `abstract` here, I beleive
Otherwise may will forgot to override

Reply:
I do not really know if we will need to override it somewhere, because it was added only to have the opportunity to compute boolean formulas for guard functions